src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -217,6 +236,22 @@ the event.  For instance if an exception was captured the hint might carry the
 original exception object.  Not all SDKs are required to provide this.  The
 parameter however is reserved for this purpose.
 
+## Event Pipeline
+
+Any event that is captured by `capture_event` is processed in the following
```

Comment:
Strictly editorial, to tighten up the language: "An event captured by `capture_event` processes in the following order: **Note:** The event can be discarded at any of these stages, at which point it is no longer processed."

Reply:
TBH, I think active voice is not the right thing here. The event does not process itself ;-)